packages/ui-i18n/src/bidirectional.tsx

```diff
@@ -32,6 +32,7 @@ import hoistNonReactStatics from 'hoist-non-react-statics'
 // see https://github.com/microsoft/TypeScript/issues/4881
 export type BidirectionalProps = {
   dir?: 'ltr' | 'rtl'
+  children?: React.ReactNode
```

Comment:
React's `PropsWithChildren` type can be used too:

```tsx
export type BidirectionalProps = PropsWithChildren<{
  dir?: 'ltr' | 'rtl'
}>
```

Reply:
fixed, thanks, I did not knew about this type